plugins/scaffolder-actions/scaffolder-backend-module-http-request/src/actions/run/helpers.ts

```diff
@@ -81,3 +81,12 @@ export const http = async (
   }
   return { code: res.status, headers, body };
 };
+
+export const getObjFieldCaseInsensitively = (obj = {}, fieldName: string) => {
```

Comment:
Why is the `obj` defaulted?

Reply:
To prevent unexpected errors were in runtime you can receive `undefined`.